Makefile

```diff
@@ -0,0 +1,10 @@
+APP ?= go-skeleton
+
+build:
+	@GO111MODULE=on go build -o ${APP}
+
+run: build
```

Comment:
Why don't we use `go vet` and `go impoort` before `build`

Reply:
Done.